AnnService/inc/Core/SPANN/ExtraFullGraphSearcher.h

```diff
@@ -656,6 +662,8 @@ namespace SPTAG
                         postingListBytes[i] = postingListSize[i] * vectorInfoSize;
                     }
                 }
+
+                if (p_opt.m_ssdIndexFileNum > 1) selections.SaveBatch();
```

Comment:
why move this line after fullVectors read? This causes the memory overflow issue in the batch build.

Reply:
Fixed